src/extension/providers/providerBase.ts

```diff
@@ -8,11 +8,20 @@ export default class ProviderBase {
   private _scope: string;
 
   constructor(scope: string) {
-    this.enabled = false;
+    this.enabled = this.loadEnabledState();
     this._eventEmitter = new EventEmitter();
     this._scope = scope;
   }
 
+  private loadEnabledState(): boolean {
+    const storedEnabled = localStorage.getItem("enabled");
+    return storedEnabled === "true";
+  }
+
+  private saveEnabledState(enabled: boolean): void {
+    localStorage.setItem("enabled", enabled.toString());
```

Comment:
I think "enabled" as a key is quite unsafe as any site can write this value. If we decide to use localStorage, we should prefix Alby-specific keys.

Also, we need a different key per provider, right?

For example `LBE-${this.scope}-enabled`?

Reply:
first try to identify and think about which local storage is used here. 

+ the website/inpage script context?
+ the conent script context?
+ the background script context?

we must understand where this is executed and what scope/access it has.

Reply:
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/70a6fd36-40c5-4e6b-862b-434d08eff314)


it's stored in website-specific local storage. which is accessible by in page script and content script as well. and hence from content script it can be accessed by background script also

Reply:
resolved